asciidoc/a2x.py

```diff
@@ -318,51 +326,53 @@ def exec_xsltproc(xsl_file, xml_file, dst_dir, opts=''):
         shell_cd(cwd)
 
 
-def get_source_options(asciidoc_file):
+def get_source_options(asciidoc_file: str) -> List[str]:
     '''
     Look for a2x command options in AsciiDoc source file.
     Limitation: options cannot contain double-quote characters.
     '''
-    def parse_options():
```

Comment:
Not sure why this was added as a nested function, but easy enough to just inline the code, which simplifies it, and also avoids a weirdness where `options` and `result` were defined _after_ the function, which worked due to how python local scopes things, but is just a weird quirk I'd rather avoid...